laboratory/tests/shared/pages/WalletPage.ts

```diff
@@ -0,0 +1,78 @@
+/* eslint-disable no-await-in-loop */
+import type { Locator, Page } from '@playwright/test'
+import type { SessionParams } from '../types'
+import { LOCAL_WALLET_URL } from '../constants'
+
+export class WalletPage {
+  private readonly baseURL = LOCAL_WALLET_URL
+
+  private readonly goToAccounts: Locator
```

Comment:
Minor readability, `goto` is easier to read than `goTo` imo and matches Playwright's `page.goto()` function.

```suggestion
  private readonly gotoAccounts: Locator
```